app/views/registered_applications/_event_display.html.erb

```diff
@@ -6,13 +6,14 @@
           <th>Event</th>
           <th>Count</th>
         </tr>
-        <% i = 0 %>
+        <% i = 1 %>
         <% event_counts.each do |event_name, count| %>
-          <tr id="event-<%= i %>" >
+          <tr style="background-color:<%= EventHelper.select_color(i) %>" >
             <td><%= event_name %></td>
             <td><%= count %></td>
           </tr>
           <% i += 1 %>
+          <% i = 1 if i > 31 %>
```

Comment:
It isn't clear what `31` means (its a [magic number](https://eliotsykes.com/magic-numbers)). 

Consider moving as much logic and magic numbers for calculating indexes into the `EventHelper#select_color` method as possible. `event_counts.each_with_index` may help.